docs/user/troubleshooting-guides/03-60-apirule-conversion-webhook-cert-issue.md

```diff
@@ -0,0 +1,26 @@
+# APIRule Conversion Webhook Not Working Because of Certificate Issue
+
+## Symptoms
+
+- There is an error about the conversion webhook certificate when fetching or creating an `APIRule` resource, for example:
+```bash
+Error from server: conversion webhook for gateway.kyma-project.io/v1beta1, Kind=APIRule failed: Post "https://api-gateway-webhook-service.kyma-system.svc:9443/convert?timeout=30s": x509: certificate has expired or is not yet valid
+```
```

Comment:
```suggestion
  ```bash
  Error from server: conversion webhook for gateway.kyma-project.io/v1beta1, Kind=APIRule failed: Post "https://api-gateway-webhook-service.kyma-system.svc:9443/convert?timeout=30s": x509: certificate has expired or is not yet valid
  ```
```